docs/content/concepts/terminology.md

```diff
@@ -1,100 +1,142 @@
+controller will target the virtual workspace instead of each specific workspace).
 
-An index is the authoritative list of a particular API in their source of truth across the system.  For instance, in
-order for a user to see all the workspaces they have available, they must consult the workspace index to return a list
-of their workspaces.  It is expected that indices are suitable for consistent LIST/WATCHing (in the kubernetes sense) so
-that integrations can be built to view the list of those objects.
+An API object has one source of truth, i.e. it's transactional stored in one system. In the context of kcp, this
+means that an API object is stored in one workspace/logical cluster. However, a use case that arises is how can someone
+view API objects across different workspaces in a "single pane of glass" fashion. Concretely:
 
-Index in the control plane sense should not be confused with secondary indices (in the database sense), which may be
-used to enable a particular index.
+- the user may have access to different workspaces: a workspace that belongs to the user personally, or one that
+  belongs to a business organization. User might want to list and manage their API objects from all workspaces in a
+  single call (e.g. single `kubectl get` invocation),
+- Service Provider might want to get a list of all instances (from all users) of the API resource they provide, so that
+  they can do reconciliation, have insights into how their API is used, and more.
+
+More information, including concrete examples and a list of frequently asked questions, can be found in the
+[Virtual Workspaces document](../workspaces/virtual-workspaces/).
+
+## Exporting/Binding APIs
+
+One of the core values of the kcp project is to enable providing APIs that can be consumed by multiple workspaces.
+This is done via exporting/binding mechanism, i.e. via API Exports and API Bindings.
+
+The general workflow is:
+
+- Define your API resources similar to how you would define CustomResourceDefinitions (CRDs)
+- Create an API Export (in the same workspace) with a list of API resources that you want to export
+- User can create an API Binding in their workspaces referring to your API Export. This will result in API resources
+  that are defined in the referenced API Export to be installed in user's workspace
+- User can now create API resources of those types in their workspace. You can build and run controllers that are going
+  to reconcile those resources across different workspaces.
+
+The Workspace model defines one particular implementation of the lifecycle of a logical cluster and the APIs within it.
+Because APIs and the implementations that back an API evolve over time, it is important that the binding be
+introspectable and orchestrate-able - that a consumer can provide a rolling deployment of a new API or new
+implementation across hundreds or thousands of workspaces. The evolution of an API within a workspace and across
+workspaces is of key importance.
```

Comment:
this paragraph is half a vision than what we have implemented actually.

Reply:
I know, and I was torn on deleting it, but I mostly didn't because I wasn't sure if that would be okay. I'd propose that we either delete it, or put it a note block and say our vision for this is to allow ... (what's written there). Which one do you prefer?

IMO, because this is supposed to be an entry-level document, I'd keep vision out from it, and focus only on what we already have. Maybe we can put it in some other document, I think it's already preset somewhere.